<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgConnector</class>
 <widget class="QDialog" name="dlgConnector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Establish new postgreSQL connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbxConnDet">
       <property name="title">
        <string>PostgreSQL connection credentials</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblConnName">
          <property name="text">
           <string>Connection name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledConnName">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>e.g. myconnection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblHost">
          <property name="text">
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledHost">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>e.g. 127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPort">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledPort">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>e.g. 5432</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDb">
          <property name="text">
           <string>Database:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledDb">
          <property name="placeholderText">
           <string>e.g. mydatabase</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblUserName">
          <property name="text">
           <string>User name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledUserName">
          <property name="placeholderText">
           <string>e.g. postgres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPassw">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsPasswordLineEdit" name="qledPassw"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Save credentials</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
